doltpy/cli/dolt.py

```diff
@@ -282,7 +266,7 @@ def add(self, table_or_tables: Union[str, List[str]]):
             to_add = [table_or_tables]
         else:
             to_add = table_or_tables
-        self.execute(["add"] + to_add, restart_server=True)
```

Comment:
have you considered using black for consistent formatting, and pre-commit hooks for other details like import ordering, etc?

Reply:
What's black?

Reply:
https://github.com/psf/black

Reply:
That sounds like a great idea!